README.rst

```diff
@@ -193,15 +193,75 @@ Add the following settings to ``settings.py``:
+|              |                                                                    |
+|              |   (                                                                |
+|              |     ('openwisp_controller.connection.connectors.ssh.Ssh', 'SSH'),  |
+|              |   )                                                                |
++--------------+--------------------------------------------------------------------+
+
+Available connector classes. Connectors are python classes that specify ways
+in which OpenWISP can connect to devices in order to launch commands.
+
+``OPENWISP_UPDATE_STRATEGIES``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+----------------------------------------------------------------------------------------+
+| **type**:    | ``tuple``                                                                              |
++--------------+----------------------------------------------------------------------------------------+
+| **default**: | .. code-block:: python                                                                 |
+|              |                                                                                        |
+|              |   (                                                                                    |
+|              |     ('openwisp_controller.connection.connectors.openwrt.ssh.OpenWrt', 'OpenWRT SSH'),  |
+|              |   )                                                                                    |
++--------------+----------------------------------------------------------------------------------------+
+
+Available update strategies. An update strategy is a subclass of a
+connector class which defines an ``update_config`` method which is
+in charge of updating the configuratio of the device.
+
+This operation is launched in a background worker when the configuration
+of a device is changed.
+
+It's possible to write custom update strategies and add them to this
+setting to make them available in OpenWISP.
+
+``OPENWISP_CONFIG_UPDATE_MAPPING``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+--------------------------------------------------------------------+
+| **type**:    | ``dict``                                                           |
++--------------+--------------------------------------------------------------------+
+| **default**: | .. code-block:: python                                             |
+|              |                                                                    |
+|              |   {                                                                |
+|              |     'netjsonconfig.OpenWrt': OPENWISP_UPDATE_STRATEGIES[0][0],     |
+|              |   }                                                                |
++--------------+--------------------------------------------------------------------+
+
+A dictionary that maps configuration backends to update strategies in order to
```

Comment:
`A dictionary which maps` instead of `A dictionary that maps`. Just a language thing. Noticed the same thing some lines above, too.